### PR TITLE
[ETCM-464] Error handling for rate-limited requests

### PR DESCRIPTION
**Pull request by conqeror**
Switched to using `fetch` for call to faucet to enable better error handling.
Display proper error message with backoff seconds from BE in case that user is rate-limited.
Update webpack config with current faucet address.

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -1,36 +1,69 @@
...
-      messageAddress.textContent = `Address: ${address}`
-      messageTxId.textContent = `Transaction ID: ${result}`
-      successMessage.style.display = 'block';
-    })
-    .catch((err) => {
+  const requestBody = {
+   	"jsonrpc": "2.0",
+    "method": "faucet_sendFunds", 
+    "params": [address],
+    "id": 1
```
Comment (qwhex) [inline src/js/index.js]: Are you using prettier?

File: src/js/index.js [hunk 2]
```diff
@@ -1,36 +1,69 @@
...
+  }
+
+  let headers = new Headers();
```
Comment (qwhex) [inline src/js/index.js]: Isn't `const` fine?

File: src/js/index.js [hunk 3]
```diff
@@ -1,36 +1,69 @@
...
 const submitButton = document.getElementById('form-submit');
 
-requestForm.onsubmit = (event) => {
+const formatBackoffTime = (seconds) => formatDuration(intervalToDuration({ start: 0, end: seconds * 1000 }))
+
+requestForm.onsubmit = async (event) => {
   event.preventDefault();
   successMessage.style.display = 'none';
   errorMessage.style.display = 'none';
+  rateLimitedErrorMessage.display = 'none';
```
Comment (qwhex) [inline src/js/index.js]: Extract these as `resetMessages()`?

File: src/js/index.js (hunk 1 above)
Comment (conqeror) [reply src/js/index.js]: I will add it :) The goal was to have as minimal project as possible here, but it's just tedious to format all the code manually :)

**Review by qwhex — APPROVED** [review]

**Pull request merged**